packages/atlas/src/hooks/useDisplaySignInDialog.tsx

```diff
@@ -2,23 +2,42 @@ import styled from '@emotion/styled'
 import React, { useCallback } from 'react'
 
 import { Text } from '@/components/Text'
-import { SvgSignUpDialogIllustration } from '@/components/_illustrations'
+import { SvgSignUpDialogIllustration, SvgSignUpDialogPattern } from '@/components/_illustrations'
 import { useConfirmationModal } from '@/providers/confirmationModal'
-import { media, sizes } from '@/styles'
+import { cVar, media, sizes } from '@/styles'
 
 const SignInDialogcontent = () => {
   return (
     <>
-      <StyledSvgSignUpDialogIllustration />
+      <IllustrationWrapper>
+        <StyledSvgSignUpDialogIllustration />
+        <StyledSvgSignUpDialogPattern />
```

Comment:
Can we just regenerate only these two illustrations and just ignore the others? Just don't commit them. We will need to probably rebase anyway after https://github.com/Joystream/atlas/pull/2457 is merged.

Reply:
there you go